src/feature_engg_scripts.py

```diff
@@ -0,0 +1,117 @@
+
+import pandas as pd
+from src.utils import (read_from_s3,read_data,upload_to_s3,authenticate_s3)
+
+client, bucket_name = authenticate_s3(which_bucket= "raw_data_bucket_name")
+
+
+df_master = read_from_s3(filename = 'final_master_agg_file',which_bucket= "raw_data_bucket_name")
+df_master.head()
+
+# df_master = pd.read_csv("E:/E2E_DTP_Project/data/final_master_agg_file.csv")
+
+upload_to_s3(df = read_data("E:/E2E_DTP_Project/data/sales.csv"), filename = "sales",which_bucket = "raw_data_bucket_name")
```

Comment:
why do you have this code here?

Reply:
Because we need product id, category and unit price columns 
It has connect with line 42 to line 51 in the same code